src/components/ui/QuickSearchButton.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { action } from '@storybook/addon-actions'
+
+import QuickSearchButton from './QuickSearchButton'
+
+const Body = ({ children }) => (
+  <div style={{ width: '200px', padding: '10px 0', border: '1px solid black' }}>
+    {children}
+  </div>
+)
+
+storiesOf('UI Components/QuickSearchButton', module)
+  .add('basic usage', () => <Body><QuickSearchButton onClick={action('onClick')} /></Body>)
+  .add('missing props (does component explode?)', () => <Body><QuickSearchButton /></Body>)
```

Comment:
This suggestion is because ideally we want to make it clear the showcases that are intended for Debug. For a context, consider that we have different types of users in Storybook, we the developers are interested on all showcases, the default ones and the debug ones, but there's also designers and product users that will be using the tool, and for them the Debug type of showcases are not useful at all, so we keep it under a category for that reason.

Also, the `missing props (does component explode?)` should declare only the minimal render, no props and no wrappers.

```suggestion
storiesOf('UI Components/QuickSearchButton/Debug', module)
  .add('missing props (does component explode?)', () => <QuickSearchButton />)
```